src/endpoints/users-endpoint.ts

```diff
@@ -0,0 +1,26 @@
+import { Endpoint, Request, Response } from 'death-adder';
+
+class UsersEndpoint extends Endpoint {
+  // get all users
+  async get(request: Request, response: Response) {
+    const users = await request.attachments.db.query(
+      'SELECT username FROM users;'
+    );
+
+    response.json(users);
+  }
+
+  // update a user
+  async put(request: Request, response: Response) {
```

Comment:
Please use multi line comments for class methods and additionally please refer to https://tsdoc.org/ for a lot of information on standardizing your comments. Information about the shape of the payload for a request to this endpoint wouldn't hurt either.